drracket-tool-text-lib/drracket/private/syncheck/traversals.rkt

```diff
@@ -48,76 +48,81 @@
-                                     phase-to-varrefs
-                                     phase-to-varsets
-                                     phase-to-tops
-                                     binding-inits
-                                     templrefs
-                                     module-lang-requires
-                                     phase-to-requires
-                                     sub-identifier-binding-directives)
-                     (annotate-variables user-namespace
-                                         user-directory
-                                         phase-to-binders
-                                         phase-to-varrefs
-                                         phase-to-varsets
-                                         phase-to-tops
-                                         templrefs
-                                         module-lang-requires
-                                         phase-to-requires
-                                         sub-identifier-binding-directives)
-                     (annotate-contracts sexp
-                                         (hash-ref phase-to-binders 0 (λ () (make-id-set 0)))
-                                         (hash-ref binding-inits 0 (λ () (make-id-set 0)))
-                                         binder+mods-binder)
-                     (when print-extra-info?
-                       (print-extra-info (list (list 'phase-to-binders phase-to-binders)
-                                               (list 'phase-to-varrefs phase-to-varrefs)
-                                               (list 'phase-to-varsets phase-to-varsets)
-                                               (list 'phase-to-tops phase-to-tops)
-                                               (list 'phase-to-requires phase-to-requires)
-                                               (list 'binding-inits binding-inits)
-                                               (list 'templrefs templrefs)
-                                               (list 'module-lang-requires module-lang-requires)
-                                               (list 'requires requires)
-                                               (list 'require-for-syntaxes require-for-syntaxes)
-                                               (list 'require-for-templates require-for-templates)
-                                               (list 'require-for-labels require-for-labels)
-                                               (list 'sub-identifier-binding-directives
-                                                     sub-identifier-binding-directives)))))]
-                  [else
+              (define is-module?
+                (syntax-case sexp (module)
+                  [(module . rest
+                     )
+                   #t]
+                  [_ #f]))
+              (cond
+                [is-module?
+                 (let ([phase-to-binders (make-hash)]
```

Comment:
How come this let didn't go away?

Reply:
Resyntax hit the 20 fix limit for the pull request and decided not to do any more work, I believe.